frontend/www/js/omegaup/typeahead.ts

```diff
@@ -100,6 +100,43 @@ function typeahead<T extends { label: string; value: string }>(
     .trigger('change');
 }
 
+export function nominationTypeahead(
+  elem: JQuery<HTMLElement>,
+  cb?: CallbackType<types.NominationListItem>,
```

Comment:
Esta función podría recibir otro parámetro con la información de la columna a filtrar:
```suggestion
  elem: JQuery<HTMLElement>,
  column: string,
  cb?: CallbackType<types.NominationListItem>,
```